operatorcert/entrypoints/update_cert_project_data.py

```diff
@@ -0,0 +1,80 @@
+"""
+Script for updating the cert project related data
+"""
+import argparse
+import logging
+from typing import Any
+from datetime import datetime, timezone
+from urllib.parse import urljoin
+
+from operatorcert import pyxis
+from operatorcert.utils import store_results
+
+LOGGER = logging.getLogger("operator-cert")
+
+
+def setup_argparser() -> argparse.ArgumentParser:
+    """
+    Setup argument parser
+
+    Returns:
+        Any: Initialized argument parser
+    """
+    parser = argparse.ArgumentParser(
+        description="Get the Certification Project related data"
+    )
+    parser.add_argument(
+        "--cert-project-id",
+        help="Identifier of certification project from Red Hat Connect",
+    )
+    parser.add_argument(
+        "--pyxis-url",
+        default="https://pyxis.engineering.redhat.com/",
+        help="Base URL for Pyxis container metadata API",
+    )
+    parser.add_argument(
+        "--certification-status",
```

Comment:
Could you add a list of valid choices, like it was done here? 
https://github.com/redhat-openshift-ecosystem/operator-pipelines-images/pull/33/files#diff-6fd4ec5f55fb2b9379b3ff244d46baa31a77a36b887306ace3d3f559d2e57a91R17